package.json

```diff
@@ -33,6 +33,7 @@
     "samsam": "1.1.2"
   },
   "devDependencies": {
+    "bluebird": "3.4.1",
```

Comment:
No points in adding this to the environment. We have checks to see if the environment supports Promises and if not the tests will not execute.